core/services/s4/postgres_orm.go

```diff
@@ -71,7 +71,7 @@ RETURNING id;`, o.tableName)
 	if errors.Is(err, sql.ErrNoRows) {
 		return ErrVersionTooLow
 	}
-	return nil
+	return err
```

Comment:
Is it possible to trigger as error here in a test? It looks like it should never happen now...

Reply:
It will be hard to trigger, e.g. close DB connection..